ydb/core/grpc_services/local_rpc/local_rpc.h

```diff
@@ -293,23 +293,23 @@ concept TRequestWithOperationParams = requires(TRequest& request) {
 };
 
 template<TRequestWithOperationParams TRequest>
-void SetRequestSyncOperationMode(TRequest& request) {
-    request.mutable_operation_params()->set_operation_mode(Ydb::Operations::OperationParams::SYNC);
+void SetRequestSyncOperationMode(TRequest& request, Ydb::Operations::OperationParams::OperationMode operationMode = Ydb::Operations::OperationParams::SYNC) {
```

Comment:
I think it was intented to override template instantiation for specific request to change sync mode of an operation